acid/features/history/model.py

```diff
@@ -106,7 +106,12 @@ def build_number(self):
         try:
             return int(self.log_url.split('/')[-3])
```

Comment:
I'd suggest:

`
split_url = self.log_url.split('/')

build_number = int(split_url[-3]) if split_url[-3].isdigit() else int(split_url[-4])

return build_number
`

Reply:
Changed to be more readable, although without isdigit(), please review.

Reply:
Done with isdigit(), sigh.